### ISSUE TITLE
CTRL+SHIFT+E is a bad choice of a keyboard shortcut

### ISSUE THREAD

**Issue by tamius-han** [ryanSN/winmoji#184]
`CTRL+SHIFT+E` conflicts with shortcuts for semi-common actions for quite some programs. For example, that's the "export (as)" shortcut in:

* GIMP
* Audacity
* Inkscape (export to PNG)
* Figma

And also

* Krita: Flatten layers
* VSCode: toggles explorer view and switches focus

(And that's just the ones I use on at least a semi-regular basis — ctrl+shift+e seems to be used either for export or merge layers in a lot of image editing programs).

This makes `CTRL + SHIFT + E` a rather poor choice for a global keyboard shortcut.

Better options:
* `WIN + .` sounds like a rather sensible option. At worst, it would override Windows' default emoji picker (for versions of windows that have it), which is not a big loss since Windows' default emoji picker isn't terribly functional in the first place (doesn't do search)
* Let user decide
* Both of the above

**Comment by displayerror** [comment]
I use AutoHotkey for custom WinKey+X combinations for launching apps - so I've had to disable Windows' built-in shortcuts, hence the need for a third party emoji tool. Now I find winmoji is interfering with my workflow.

I use `CTRL + SHIFT + E` for a lot of programs. These are mainly drawing/painting and 3D applications. I hold a pen stylus in my right hand, which causes most of my shortcuts to be heavily focused around the bottom left modifiers, so `CTRL + SHIFT + E` gets used a lot.

Is there any way to disable it? Or better yet, allow the user to set a custom shortcut?

**Comment by ryanSN** [comment]
Yes totally. This has always been on my radar and the current shortcut was a stop gap. The intent was to eventually have a settings section  to allow this as well set some other items such as run on startup, always on top. To name a few options around this idea. 

Sorry for the delayed response on this. A quick workaround solution would be to allow the editing of short cut manually in the local settings file. Would that work for some? For now? I may already expose it there and will have to double check.

**Comment by mbomb007** [comment]
> * `WIN + .` sounds like a rather sensible option. At worst, it would override Windows' default emoji picker (for versions of windows that have it), which is not a big loss since Windows' default emoji picker isn't terribly functional in the first place (doesn't do search)

Maybe Windows didn't used to have search, but it does now.

> Yes totally. This has always been on my radar and the current shortcut was a stop gap. The intent was to eventually have a settings section to allow this as well set some other items such as run on startup, always on top. To name a few options around this idea.
> 
> Sorry for the delayed response on this. A quick workaround solution would be to allow the editing of short cut manually in the local settings file. Would that work for some? For now? I may already expose it there and will have to double check.

What local settings file? Also, Ctrl+Shift+E doesn't even work.

**Comment by mbomb007** [comment]
Also, the WinMoji search doesn't work well because it also includes fuzzy matches. So the GIF example showing only fish when searching "fish" isn't accurate.
